### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by vjtknn**
# Adds:
- comment model
- association comments with user and book
- specs for comment model

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+
+  def create
+    @comment = Comment.create(comment_params)
+    @comment.user = current_user
+    if @comment.save
+      redirect_to @comment.book
+    end
```
Comment (Qsmir) [inline app/controllers/comments_controller.rb]: missing else?

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
+  def destroy
+    @comment = Comment.find(params[:id])
+    @comment.destroy
```
Comment (Qsmir) [inline app/controllers/comments_controller.rb]:
```
if @comment.destroy
  ...
else 
  ...
end
```

File: spec/forms/books_form_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# require 'rails_helper'
```
Comment (Qsmir) [inline spec/forms/books_form_spec.rb]: comments ? :)

**Comment by vjtknn** [comment]
comments

**Pull request merged**